language/stdlib/modules/TravelRuleLimit.move

```diff
@@ -0,0 +1,39 @@
+address 0x0 {
+
+    module TravelRuleLimit {
+        use 0x0::LibraConfig;
+        use 0x0::Signer;
+        use 0x0::Transaction;
+        use 0x0::Association;
+
+        struct TravelRuleLimit {
+            micro_libra_limit: u64,
+        }
+
+        /// Travel rule limit set during genesis
+        public fun initialize(account: &signer) {
+            Transaction::assert(Signer::address_of(account) == LibraConfig::default_config_address(), 1);
+
+            LibraConfig::publish_new_config<TravelRuleLimit>(
+                account,
+                TravelRuleLimit { micro_libra_limit: 1000000 },
+            );
+        }
+
+        // Anyone can fetch the current set limit
+        public fun get_cur_limit(): u64 {
```

Comment:
Might be worth including "micro" in the name to avoid the expectation that this is 1000...